content/docs/1.36/deployment.md

```diff
@@ -653,7 +653,7 @@ You can find more information about topics and partitions in general in the [off
 
 Jaeger supports an extension mechanism that allows the storage to be implemented as a gRPC server. The server can run either as a child process of Jaeger components (the Hashicorp go-plugin model), or as a remote gRPC service (since Jaeger v1.30). For more information, please refer to [jaeger/plugin/storage/grpc](https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/grpc).
 
-Available plugins:
+#### Available plugins:
```

Comment:
```suggestion
- Available sidecar plugins:
```
let's not add headers because the context menu on the right is already very long